### DISCUSSION TITLE
Could you make eaglercraft forge on web?

### DISCUSSION THREAD

**Discussion by PengiCoder** [eaglerforge/EaglerForge#25] [Ideas]
Or is that a "never implementing" thing

**Comment by stpv221** [comment]
WDYM

**Comment by leahanderson1** [comment]
wtf do you mean
are you looking for HTML builds?
if so https://github.com/EaglerForge/Eaglerforge-builds

**Comment by PengiCoder** [comment]
Nvm

**Comment by sillyangel** [comment]
I think he ment about adding REAL forge on the web using eaglercraft
